posts/rotating-ssh-keys.md

```diff
@@ -20,19 +20,17 @@ Copy your existing SSH folder to temporary folder for safekeeping.
 
 ## Create a strong passphrase
 
-Generate a strong passphrase for your new SSH key. I use [1Password](https://agilebits.com/onepassword) to generate a nice, long passphrase with at least 2 digits and 2 symbols.
+Generate a strong passphrase for your new SSH key. I use [1Password](https://agilebits.com/onepassword) to generate a nice, long passphrase.
 
 ## Create a new key pair
 
-Create your new SSH key pair. If you already have a key named "id_rsa", this will overwrite it.
+Create your new SSH key pair. If you already have a key named `id_rsa`, this will overwrite it. When prompted, enter the passphrase you generated earlier
```

Comment:
Should there be a period at the end of this sentence?

Reply:
Yep - thank you!

Reply:
Fixed in #453!